c_glib/arrow-cuda-glib/meson.build

```diff
@@ -17,88 +17,91 @@
-                             implicit_include_directories: false,
-                             include_directories: base_include_directories,
-                             cpp_args: ['-DGARROW_CUDA_COMPILATION'],
-                             soversion: so_version,
-                             version: library_version)
-arrow_cuda_glib = declare_dependency(link_with: libarrow_cuda_glib,
-                                     include_directories: base_include_directories,
-                                     dependencies: dependencies)
+dependencies = [arrow_cuda, arrow_glib]
+libarrow_cuda_glib = library(
+    'arrow-cuda-glib',
+    sources: sources,
+    install: true,
+    dependencies: dependencies,
+    implicit_include_directories: false,
+    include_directories: base_include_directories,
+    cpp_args: ['-DGARROW_CUDA_COMPILATION'],
+    soversion: so_version,
+    version: library_version,
+)
+arrow_cuda_glib = declare_dependency(
+    link_with: libarrow_cuda_glib,
+    include_directories: base_include_directories,
+    dependencies: dependencies,
+)
 if target_machine.system() != 'windows'
-  pkgconfig.generate(libarrow_cuda_glib,
-                     description: 'C API for Apache Arrow CUDA based on GLib',
-                     filebase: 'arrow-cuda-glib',
-                     name: 'Apache Arrow CUDA GLib',
-                     requires: ['arrow-glib', 'arrow-cuda'],
-                     variables: pkgconfig_variables,
-                     version: version)
+    pkgconfig.generate(
+        libarrow_cuda_glib,
+        description: 'C API for Apache Arrow CUDA based on GLib',
+        filebase: 'arrow-cuda-glib',
+        name: 'Apache Arrow CUDA GLib',
+        requires: ['arrow-glib', 'arrow-cuda'],
+        variables: pkgconfig_variables,
+        version: version,
+    )
 endif
 
 if have_gi
-  gir_dependencies = [
-    declare_dependency(sources: arrow_glib_gir),
-  ]
-  gir_extra_args = [
-    '--warn-all',
-    '--include-uninstalled=./arrow-glib/Arrow-1.0.gir',
-  ]
-  arrow_cuda_glib_gir = \
-    gnome.generate_gir(libarrow_cuda_glib,
-                       dependencies: gir_dependencies,
-                       export_packages: 'arrow-cuda-glib',
-                       extra_args: gir_extra_args,
-                       header: 'arrow-cuda-glib/arrow-cuda-glib.h',
-                       identifier_prefix: 'GArrowCUDA',
-                       includes: [
-                         'Arrow-1.0',
-                       ],
-                       kwargs: generate_gi_common_args,
-                       namespace: 'ArrowCUDA',
-                       sources: sources + c_headers,
-                       symbol_prefix: 'garrow_cuda')
+    gir_dependencies = [declare_dependency(sources: arrow_glib_gir)]
+    gir_extra_args = [
+        '--warn-all',
+        '--include-uninstalled=./arrow-glib/Arrow-1.0.gir',
+    ]
+    arrow_cuda_glib_gir = \
+    gnome.generate_gir(
```

Comment:
Hmm. This is strange format...

Could you try removing the `\` and a newline after it?

Reply:
Nice catch - done!